de-lib/src/main/java/org/iplantc/de/client/services/DiskResourceServiceFacade.java

```diff
@@ -144,29 +146,28 @@ void moveContents(final Folder sourceFolder,
      * @param callback callback executed when service call completes.
      */
     <T extends DiskResource> void deleteDiskResources(List<T> diskResources,
-                                                      AsyncCallback<HasPaths> callback);
+                                                      DECallback<HasPaths> callback);
 
     /**
      * Call service to delete disk resources in case user selects all items
      * 
      * @param selectedFolderId the folder whose contents will be deleted.
      */
-    void deleteContents(String selectedFolderId, AsyncCallback<HasPaths> callback);
+    void deleteContents(String selectedFolderId, DECallback<HasPaths> callback);
 
     /**
      * Call service to delete disk resources (i.e. {@link File}s and {@link Folder}s)
      * 
      * @param diskResources a set of <code>DiskResource</code>s to be deleted
      * @param callback callback executed when service call completes.
      */
-    void deleteDiskResources(HasPaths diskResources, AsyncCallback<HasPaths> callback);
+    void deleteDiskResources(HasPaths diskResources, DECallback<HasPaths> callback);
 
     /**
      * @param resource the <code>DiskResource</code> for which metadata will be retrieved.
      * @param callback callback executed when service call completes.
      */
-    void getDiskResourceMetaData(DiskResource resource,
-                                 AsyncCallback<DiskResourceMetadataList> callback);
+    void getDiskResourceMetaData(DiskResource resource, DECallback<DiskResourceMetadataList> callback);
```

Comment:
I think for now, I'll switch the get and set metadata endpoints back to the old `AsyncCallback` so they can't disable the Data window.